lighthouse-core/audits/content-width.js

```diff
@@ -55,8 +55,8 @@ class ContentWidth extends Audit {
       return '';
     }
 
-    return 'The content scroll size is ' + artifact.scrollWidth + 'px, ' +
-        'whereas the viewport size is ' + artifact.viewportWidth + 'px.';
+    return 'The content scroll size is ' + artifact.innerWidth + 'px, ' +
+        'whereas the viewport size is ' + artifact.outerWidth + 'px.';
```

Comment:
No worries, and I was just going with the "if it ain't broke don't fix it" approach, but I agree `documentElement.scrollWidth !== documentElement.clientWidth` makes the intention a lot more clear. In practice, outerWidth seems to reflect the non-scaled true width of the viewport and innerWidth seems to reflect the width of the content